### PR TITLE
Add common properties to entities after transformation

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
Like it says on the tin, this adds some common properties to entities after transformation. This is to reduce the boilerplate needed.

**Note:** This does more than the connected ticket asked for. :man_shrugging: 

## Testing done
`node script/cms/test-bundle-transformers.js` transforms all the entities successfully.

## Acceptance criteria
- [x] The `entityUrl` and some other properties are added to every entity after transformation _if_ the properties didn't exist on the entity already

### CODE REVIEW

File: src/site/stages/build/process-cms-exports/helpers.js [hunk 1]
```diff
@@ -41,6 +41,33 @@ function getContentModelType(entity) {
   return [entity.baseType, subType].filter(x => x).join('-');
 }
 
+/**
+ * Add common properties to the entity. Does not mutate `entity`; instead,
+ * returns a new object.
```
Comment (bkjohnson) [inline src/site/stages/build/process-cms-exports/helpers.js:46]: Is there an advantage here to not modifying the entity directly?

File: src/site/stages/build/process-cms-exports/index.js [hunk 2]
```diff
@@ -106,6 +106,36 @@ const validateOutput = (entity, transformedEntity) => {
...
+const addCommonProperties = (transformedEntity, originalEntity) => {
+  /* eslint-disable no-param-reassign */
+  transformedEntity.contentModelType =
+    transformedEntity.contentModelType || getContentModelType(originalEntity);
+  const [
+    entityType,
+    entityBundle,
+  ] = transformedEntity.contentModelType.includes('-')
+    ? transformedEntity.contentModelType.split('-')
+    : [transformedEntity.contentModelType, transformedEntity.contentModelType];
```
Comment (bkjohnson) [inline src/site/stages/build/process-cms-exports/index.js:125]: Having `entityType` and `entityBundle` be the same here seems odd.  For things like `paragraph-q_a_section` it's clear which is the bundle and which is the type, but for a plain ol' `paragraph` why do we need the bundle as well?

**Review by bkjohnson — COMMENTED** [review]
Has `entityUrl` been removed from the CMS?

**Review by drorva — APPROVED** [review]

File: src/site/stages/build/process-cms-exports/helpers.js (hunk 1 above)
Comment (cvalarida) [reply src/site/stages/build/process-cms-exports/helpers.js:46]: Yeah, I was concerned about the memory it'd use to create new objects for every single entity. Tens of thousands of times. Then you consider the added time for creation of those objects, and the added time for garbage collection, and a seemingly simple thing becomes kind of a pain.

Comment (cvalarida) [reply src/site/stages/build/process-cms-exports/helpers.js:46]: To be fair, I didn't run any tests. It was all assumption.

File: src/site/stages/build/process-cms-exports/index.js (hunk 2 above)
Comment (cvalarida) [reply src/site/stages/build/process-cms-exports/index.js:125]: The `entityBundle` is used to determine which Liquid template to use.

**Comment by cvalarida** [comment]
`entityUrl` has not been removed from the CMS. When we first started creating the transformers, it wasn't in the export, but was recently(ish?) added, so now we can use it directly and don't need to construct our own `entityUrl` using the `path` and whatnot.

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**